internal/identity/tinksign.go

```diff
@@ -0,0 +1,119 @@
+package identity
+
+import (
+	"bytes"
+	"fmt"
+
+	"github.com/tink-crypto/tink-go/v2/keyset"
+	"github.com/tink-crypto/tink-go/v2/signature"
+	"github.com/tink-crypto/tink-go/v2/tink"
+)
+
+var _ Signer = &TinkSigner{}
+
+type TinkSigner struct {
```

Comment:
Why wrap the tink abstractions again?

Reply:
Originally it was because of having a NoOpEncryptor equivalent, but now probably will be dropping NoOp as well as this abstraction.